src/Confluent.Kafka/Timestamp.cs

```diff
@@ -54,9 +76,14 @@ public Timestamp(DateTime dateTime, TimestampType type)
         public TimestampType Type { get; }
 
         /// <summary>
-        ///     Gets the timestamp value.
+        ///     Get the Unix millisecond timestamp.
+        /// </summary>
+        public long UnixTimestampMs { get; }
+
+        /// <summary>
+        ///     Gets the Utc DateTime corresponding to the <see cref="UnixTimestampMs"/>.
         /// </summary>
-        public DateTime DateTime { get; }
+        public DateTime DateTime => UnixTimestampMsToDateTime(UnixTimestampMs);
```

Comment:
Ok
Oh, given this, do we accept C#7 ? Do we have a convention of when using this notation? (say less than 1 line / x characters / 1 instruction ?)